templates/pulsar.service.j2

```diff
@@ -9,10 +9,12 @@ UMask=022
 Type=simple
 User={{ __pulsar_user_name }}
 Group={{ __pulsar_user_group }}
-WorkingDirectory={{ pulsar_server_dir }}
+WorkingDirectory={{ pulsar_root }}
```

Comment:
```suggestion
```

I believe `WorkingDirectory` can be dropped entirely - at least it works in my testing.

Reply:
Fine to remove it.